### ISSUE TITLE
1.3.1 - cannot search in dropdown - Undefined index: ddtype

### ISSUE THREAD

**Issue by mathieuchateau** [pluginsGLPI/fields#77]
glpi 0.90.1
field 1.3.1

When editing dropdown, doing a search generate error 500:

from php-error:

```
  *** PHP Notice(8): Undefined index: ddtype
  Backtrace :
  plugins/fields/front/commondropdown.php:4
```

In apache error log:
[Thu Oct 27 15:36:54.698985 2016] [:error] [pid 12063] [client 1.1.1.1:61488] PHP Fatal error:  Uncaught exception 'RuntimeException' with message 'Attempt to load unsecure or missing /glpi/www/glpi/files/_plugins/fields/front/.php!' in /glpi/www/glpi/plugins/fields/front/commondropdown.php:8\nStack trace:\n#0 {main}\n  thrown in /glpi/www/glpi/plugins/fields/front/commondropdown.php on line 8

**Comment by trasher** [comment]
Will be fixed in the 1.3.2 release.

**Comment by DGermany** [comment]
I'm trying to get the members of a commondropdown-field with REST/Webservice. 

URL: https://xx.xx.xx/apirest.php/commondropdown
Method: GET
Header: App-toekn: xxxx /Sessiontoken: xxxx

just become in 500 error with no other content. 

Glpi-Version: 9.1.3
